packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -99,10 +95,7 @@ export default function BookmarkFeedLayout({
         <h1 className="font-bold typo-callout">Bookmarks</h1>
       </FeedPageHeader>
       <CustomFeedHeader
-        className={classNames(
-          'mb-6 flex',
-          shouldUseListFeedLayout && !shouldUseListMode && 'px-4',
```

Comment:
Part of the reason I consolidated the variables is this. Quite difficult to understand the difference between the two.